lua/artemis.lua

```diff
@@ -211,8 +211,18 @@ local function keymap_set(modes, lhs, rhs, opts)
     end
     table.insert(args, lhs)
     if type(rhs) == 'function' then
+      local expr = 'require"artemis"._keymap[ [=[' .. lhs .. ']=] ]()'
+      local head, tail
+      if opts.expr then
+        head = "luaeval('"
+        expr = expr:gsub("'", "''") -- vim style escape of single quote
```

Comment:
Would you mind to explain the escape? I could not see the any reference.

Reply:
This behavior is explained at `:h expr-'`.
I will append it to comment.

Reply:
On the way, I fixed code to understandable.